rdwatch/core/models/annotation_exports.py

```diff
@@ -24,8 +22,44 @@ class AnnotationExport(models.Model):
         help_text='Celery Task Id',
     )
 
+    def delete(self, *args, **kwargs):
+        if self.export_file:
+            self.export_file.delete(save=False)  # Delete the file from storage
+        super().delete(*args, **kwargs)  # Call the "real" delete() method of the model
 
-@receiver(models.signals.pre_delete, sender=AnnotationExport)
+
+@receiver(models.signals.pre_delete, sender=AbstractFileExport)
 def delete_content(sender, instance, **kwargs):
     if instance.export_file:
         instance.export_file.delete(save=False)
+
+
+class AnnotationExport(AbstractFileExport):
+    configuration = models.ForeignKey(
+        to='ModelRun',
+        on_delete=models.PROTECT,
+        help_text='The hyper parameters used for the xport',
+        db_index=True,
+    )
+
+
+class AnimationSiteExport(AbstractFileExport):
+    site_evaluation = models.ForeignKey(
+        to='SiteEvaluation',
+        on_delete=models.PROTECT,
+        help_text='SiteEvaluation for Animation export',
+        db_index=True,
+    )
+    arguments = models.JSONField(null=True, default=None)
+    user = models.ForeignKey(User, on_delete=models.CASCADE, null=True, blank=True)
+
+
+class AnimationModelRunExport(AbstractFileExport):
+    configuration = models.ForeignKey(
+        to='ModelRun',
+        on_delete=models.PROTECT,
+        help_text='ModelRun for Animation export',
+        db_index=True,
+    )
+    user = models.ForeignKey(User, on_delete=models.CASCADE, null=True, blank=True)
```

Comment:
Same question as above
```suggestion
    user = models.ForeignKey(User, on_delete=models.CASCADE)
```